src/App.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import styles from "./components/PuzzleBoard/PuzzleBoard.css";
-import phrases from "./puzzleBank/phrases.js";
+import phrases from "./dataBank/phrases.js";
+import PuzzleBoard from "./components/PuzzleBoard/puzzleBoard";
+import RefreshPuzzleButton from "./components/RefreshPuzzleButton/refreshPuzzleButton";
```

Comment:
Now that you have a separate `wheelScoreValues.js`, don't forget to import it here to be used and remove the hardcoded values in `this.state`